range.c

```diff
@@ -725,7 +765,7 @@ range_bsearch(VALUE range)
 	BSEARCH(INT2FIX);
     }
 #if SIZEOF_DOUBLE == 8 && defined(HAVE_INT64_T)
-    else if (RB_FLOAT_TYPE_P(beg) || RB_FLOAT_TYPE_P(end)) {
+    else if (RB_TYPE_P(beg, T_FLOAT) || RB_TYPE_P(end, T_FLOAT)) {
```

Comment:
This block should not be part of the commit. This seems to revert part of an earlier commit that switched more code to use `RB_FLOAT_TYPE_P`.

Reply:
Corrected.